src/Controller/Backend/FileManager.php

```diff
@@ -330,12 +331,13 @@ private function getRelatedFiles(FileInterface $file)
     {
         // Match foo(_local).*(.dist)
         $base = $file->getFilename();
-        if (Str::endsWith($base, '.dist')) {
+        Path::getExtension($base);
+        if (Path::getExtension($base) === '.dist') {
             $base = substr($base, 0, -5);
         }
         $ext = pathinfo($base, PATHINFO_EXTENSION);
-        $base = Str::replaceLast(".$ext", '', $base);
-        $base = Str::replaceLast('_local', '', $base);
+        $base = Str::replaceLast($base, ".$ext", '');
+        $base = Str::replaceLast($base, '_local', '');
```

Comment:
If users don't have dist files anymore my first thought is :fire: but I'm also wondering if there are other use cases for this. There could be other places dist files are used, not just config files and not core files. I think since we have the functionality implemented it is fine to keep it.